src/mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+strict_optional = True
+no_implicit_optional = True
```

Comment:
Just curious, but why these 2 ones? I guess the question is more to @sebastian-philipp, as this is just copy-paste from his previous contribution (just checked the original PR #26715 but didn't find any rationale there).

Reply:
foo.py
```python
def f(x: int=None):
  return x + 2
```

```bash
$ mypy --no-strict-optional foo.py   # no errors
$ mypy foo.py                             
foo.py:1: error: Unsupported operand types for + ("None" and "int")
foo.py:1: note: Left operand is of type "Optional[int]"
```
`None` is a real source of bugs in Python, if you're not careful. Looks like some if it is now enabled by default.